src/components/mainLayout/curtainVk/curtainVk.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+
+import * as VKID from "@vkid/sdk";
+
+import { ICurtainVk } from "../../../types/common/ComponentsProps";
+
+export const CurtainVK = (handleError: ICurtainVk) => {
+	const floatingOneTap = new VKID.FloatingOneTap();
+	floatingOneTap
+		.render({ appName: "freenance-app", scheme: VKID.Scheme.LIGHT, lang: VKID.Languages.RUS })
```

Comment:
function handleOpenAuthCurtain() {
		floatingOneTap.render(authCurtainRenderObj);
	}
	
can you replace this line to SignInForm and SignUpForm components creating object for render props?